lib/internal/bootstrap/node.js

```diff
@@ -231,6 +231,8 @@
 
     setupAllowedFlags();
 
+    NativeModule.require('internal/tls').setupRootCertsFile();
```

Comment:
This breaks `./configure --without-ssl` builds. Not the require statement itself but the `internalBinding('crypto')` statement in the included file, it'll throw an exception.

Reply:
Would making this conditional on `Boolean(process.versions.openssl)` as test/common/util.js does for `hasCrypto` be OK, or is there a better way?

Reply:
I have placed this line in `if (process.versions.openssl) {`.

    ... or is there a better way?

I think it's ok as `lib/internal/util` also uses `process.versions.openssl` to assert crypto:
https://github.com/nodejs/node/blob/972d0beb591859a1a0df59a3d1818493a6132bf5/lib/internal/util.js#L18

Reply:
And a new test has been added to ensure that it won't throw without ssl.